doc/camera_and_viewport.md

```diff
@@ -84,3 +84,84 @@ you when changing. The `zoom` variable is immediately applied.
 When dealing with input events, it is imperative to convert screen coordinates to world coordinates
 (or, for some reasons, you might want to do the reverse). The Camera provides two functions,
 `screenToWorld` and `worldToScreen` to easily convert between these coordinate spaces.
+
+### Using the camera with the Game class
+
+If you are not using `BaseGame` but instead are using the `Game` class then you need to manage 
+calling certain camera methods yourself. Let's say we have the following game structure, and we 
+want to add the camera functionality:
+
+```dart
+class YourGame extends Game {
+  Camera? camera;
+
+  Future<void> onLoad() async {}
+
+  void render(Canvas canvas) {}
+
+  void update(double dt) {}
+}
+```
+
+We first create a new camera instance on load and assign our game as the reference:
+
+```dart
+  // ...
+  
+  Future<void> onLoad() async {
+    camera = Camera();
+
+    // This is required for the camera to work.
+    camera?.gameRef = this;
+
+    // Not required but recommend to set it now or when you set the follow target.
+    camera?.worldBounds = yourWorldBounds;
+
+    // Rest of your on load code.
+  }
+
+  // ...
+```
+
+The camera also needs to be aware of which position to follow, this can be done anywhere in your 
```

Comment:
`needs to be aware` => this is an optional feature, you can use camera for just moving, snapping, shaking (no follow)

Reply:
Updated it